server/src/main/java/org/apache/uniffle/server/ShuffleDataFlushEvent.java

```diff
@@ -90,6 +92,14 @@ public ShuffleBuffer getShuffleBuffer() {
     return shuffleBuffer;
   }
 
+  public StorageManager getStorageManager() {
```

Comment:
Why do we use storageManager as field?

Reply:
Because `selectStorageManager` will be called again In `org.apache.uniffle.server.storage.MultiStorageManager#updateWriteMetrics`, The metrics will be incorrent if we don't this.

Reply:
It's strange that the event contains storage manager for me.  Could we record the mapping information in the storage manager?

Reply:
Agree. and i just found out `org.apache.uniffle.server.storage.MultiStorageManager#updateWriteMetrics` is not referenced, maybe we don't need do anything in the method